tests/gold_tests/pluginTest/traffic_dump/replay/http3.yaml

```diff
@@ -0,0 +1,64 @@
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+meta:
+  version: "1.0"
+
+  blocks:
+  - 200_ok_response: &200_ok_response
+      server-response:
+        status: 200
+        reason: OK
+        headers:
+          fields:
+          - [ Content-Length, 16 ]
+          - [ Connection, close ]
+
+sessions:
+- protocol: [ {name: http, version: 3},
+              {name: tls, sni: www.tls.com},
+              {name: tcp},
```

Comment:
H3 over TCP?

Also, I'm not sure if it should have tls. I can see the stack has sni key, but strictly speaking, tls is not on QUIC.

Reply:
Thank you for the observations,

Good point: in this case, the tcp is not used by Proxy Verifier. It is over UDP. That is: `{name: http, version 3}` dictates the whole stack which is over QUIC over UDP. I'll update that protocol node.

The `{name: tls}` node indicates the cryptographic characteristics of QUIC (SNI in this case). Does QUIC support unencrypted traffic? I think it has to be encrypted, yes?

Reply:
Right, it's always encrypted at least on version 1.